System.Compiler/Nodes.cs

```diff
@@ -5609,6 +5609,19 @@ public Module(TypeNodeProvider provider, TypeNodeListProvider listProvider, Cust
       get{return this.metadataImportErrors;}
       set{this.metadataImportErrors = value;}
     }
+      
+    private ArrayList metadataImportWarnings;
+    /// <summary>
+    /// Holds information about non-critical errors.
+    /// For instance, lack of /names stream in the PDB is not a critical error and should be handled
+    /// gracefully.
+    /// </summary>
+    public ArrayList MetadataImportWarnings {
```

Comment:
Totally agree that for any new code this approach should be used. But I tried to stay with existing "design" that's why, IMHO, for consistency reasons, approach with backing field could be better for now. Once we'll clean up the code, all those ugly .NET 1.0 patterns will disappear:)